### PR TITLE
Update stakater vault chart

### PR DESCRIPTION
**Pull request by sabkat**

### CODE REVIEW

File: values.openshift.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# These overrides are appropriate defaults for deploying this chart on OpenShift
```
Comment (hanzala1234) [inline values.openshift.yaml:1]: why this file?

File: templates/server-route.yaml [hunk 2]
```diff
@@ -14,20 +15,25 @@ metadata:
...
     kind: Service
     name: {{ $serviceName }}
     weight: 100
   port:
     targetPort: 8200
+  {{- if $route.tls.enabled }}
   tls:
-    termination: passthrough
+    insecureEdgeTerminationPolicy: {{ $route.tls.insecureEdgeTerminationPolicy }}
+    termination: {{ $route.tls.termination }}
```
Comment (hanzala1234) [inline templates/server-route.yaml:35]: we will need to provide tls termination policy in values.yaml, seems like its  missing

File: templates/server-route.yaml [hunk 3]
```diff
@@ -14,20 +15,25 @@ metadata:
     app.kubernetes.io/name: {{ include "vault.name" . }}
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
-    {{- with .Values.server.route.labels }}
+    {{- with $route.labels }}
```
Comment (hanzala1234) [inline templates/server-route.yaml:18]: I dont see this function in helper.tpl

Comment (sabkat) [reply templates/server-route.yaml:18]: Its not a function. Its a variable defined at the top of the file taking value from Values.server.route

File: values.openshift.yaml (hunk 1 above)
Comment (sabkat) [reply values.openshift.yaml:1]: Deleting

File: templates/server-route.yaml (hunk 2 above)
Comment (sabkat) [reply templates/server-route.yaml:35]: What should I provide. The line in red ?

File: templates/server-route.yaml (hunk 3 above)
Comment (hanzala1234) [reply templates/server-route.yaml:18]: ok. thanks for explanation

**Pull request closed without merging**